src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -8,23 +8,16 @@ import { bindActionCreators } from "redux";
 import { connect } from "react-redux";
 import * as I from "immutable";
 import classnames from "classnames";
-import { createSelector } from "reselect";
 import { sortBy } from "lodash";
 
 import actions from "../../actions";
 import CloseButton from "../shared/Button/Close";
 import { endTruncateStr } from "../../utils/utils";
 import { features } from "../../utils/prefs";
 import { getFilename } from "../../utils/source";
-import {
-  getSources,
-  getSourceInSources,
-  getBreakpoints,
-  getPauseReason,
-  getTopFrame
-} from "../../selectors";
-import { isInterrupted } from "../../utils/pause";
-import { makeLocationId } from "../../utils/breakpoint";
+
+import { getBreakpointSelector } from "../../selectors/";
```

Comment:
I think the selector should make it clear what it's used for `getBreakpointsForPane` because it's different if we want them for the editor

Reply:
Thanks, the first version was `whatchamacallit` as i was not really sure what to name it :)